pixel-saver@deadalnix.me/decoration.js

```diff
@@ -122,6 +129,10 @@ function getOriginalState(win) {
 	if (win._pixelSaverOriginalState !== undefined) {
 		return win._pixelSaverOriginalState;
 	}
+
+	if (!win.decorated) {
```

Comment:
Is that true or false on windows that are undecorated because they are maximized and have the _GTK_HIDE_TITLEBAR_WHEN_MAXIMIZED flag set ?